liesel/distributions/mvn_degen.py

```diff
@@ -206,7 +206,7 @@ def __init__(
         )
 
     @classmethod
-    def from_penalty(
+    def from_penalty_variance(
```

Comment:
I would strongly prefer a solution that only adds functionality and does not change the existing API.

Reply:
I see what you mean - still, thanks for reverting back to the old name 😊